contracts/JBTiered721Delegate.sol

```diff
@@ -184,6 +186,8 @@ contract JBTiered721Delegate is JBOwnable, JB721Delegate, IJBTiered721Delegate {
     function initialize(
         uint256 _projectId,
         IJBDirectory _directory,
+        bytes4 _payMetadataDelegateId,
```

Comment:
move these into constructor such that the interfaceIds are constant across JBTiered721Delegate deploys